lib/commands/streamscreen.js

```diff
@@ -249,12 +253,15 @@ async function initGstreamerPipeline (deviceStreamingProc, deviceInfo, opts = {}
  * @throws {Error} If screen streaming has failed to start or is not supported on the host system.
  */
 commands.mobileStartScreenStreaming = async function mobileStartScreenStreaming (options = {}) {
+  this.ensureFeatureEnabled(ADB_SCREEN_STREAMING_FEATURE);
```

Comment:
Good idea 👍 
Need to update https://github.com/appium/appium-android-driver#opt-in-features-with-security-risk , too.